package.json

```diff
@@ -1,28 +1,36 @@
 {
   "private": true,
   "scripts": {
+    "buildDeps": "yarn workspace @openvaa/app-shared build && true",
     "dev": "docker compose -f docker-compose.dev.yml up -d --force-recreate --build --wait",
     "dev:attach": "docker compose -f docker-compose.dev.yml up --force-recreate --build",
     "dev:down": "docker compose -f docker-compose.dev.yml down -v --rmi all --remove-orphans",
     "dev:stop": "docker compose -f docker-compose.dev.yml stop",
     "prepare": "husky",
     "prod": "docker compose -f docker-compose.yml -p vaa-prod up -d --build",
-    "test": "playwright test"
+    "test": "yarn workspace @openvaa/tests test",
+    "format": "yarn buildDeps && prettier --write .",
+    "format:check": "yarn buildDeps && prettier --check .",
+    "lint": "yarn buildDeps && eslint --flag unstable_config_lookup_from_file backend/vaa-strapi/src backend/vaa-strapi/tests frontend packages tests",
```

Comment:
The flag's needed until eslint 10 to use nested eslint.configs. We need to enumerate the folders in the backend/vaa-strapi we want to lint because, some of the other folders contain eslint.configs that cause an error with eslint 9 – even if we mark them as ignored!